### ISSUE TITLE
Bookmarked icon keeps displayed after the bookmark is deleted

### ISSUE THREAD

**Issue by luixxiul** [brave/browser-laptop#10348]
**Describe the issue you encountered:** Bookmarked icon keeps displayed after the bookmark is moved to a bookmark folder and deleted.

<img width="395" alt="screenshot 2017-08-09 10 49 11" src="https://user-images.githubusercontent.com/3362943/29101894-69e86aa6-7cf0-11e7-837f-1adb84b1093c.png">

- Platform (Win7, 8, 10? macOS? Linux distro?): macOS

- Brave Version (revision SHA): master branch

- Steps to reproduce:
    1. Bookmark twitter.com
    2. Create a bookmark folder
    3. Move the bookmark into the folder
    4. Delete the bookmark 
    
- Actual result: the bookmarked icon is still displayed

- Expected result: it should not

- Extra QA steps:
    1.
    2.
    3.

- Any related issues:

**Comment by luixxiul** [comment]
This is the log on the devtool console

````
http://localhost:8080/gen/app.entry.js:60926: Uncaught TypeError: Cannot read property 'create' of undefined{TypeError: Cannot read property 'create' of undefined
    at EventEmitter.ipc.on (http://localhost:8080/gen/app.entry.js:60926:34)
    at emitTwo (extensions::event_emitter:105:13)
    at EventEmitter.emit (extensions::event_emitter:193:7)
    at EventEmitter.ipcRenderer.emit (extensions::ipc_utils:31:22)}
````

**Comment by luixxiul** [comment]
related: https://github.com/brave/browser-laptop/issues/10353#issue-248945834

**Comment by cezaraugusto** [comment]
@luixxiul could you confirm this is still a thing? this is very similar to #11672. 

I'm on 0.20.x and couldn't reproduce both. Also tried on 0.21.x and master, works normally.

**Comment by NejcZdovc** [comment]
this is fixed in #10310

**Comment by NejcZdovc** [comment]
but I already fixed it in dc49efa132eb509df323c57b62c85a1950fe261e which is already merged

**Comment by luixxiul** [comment]
confirmed it's been fixed 👍
